src/modules/hip/kernel/crop_mirror_normalize.hpp

```diff
@@ -1,33 +1,45 @@
 #include <hip/hip_runtime.h>
 #include "rpp_hip_common.hpp"
 
-__device__ void cmn_hip_compute(uchar *srcPtr, d_float8 *pix_f8, d_float8 *cmnParams_f8)
+__device__ void cmn_hip_compute(uchar *srcPtr, float *dstPtr, d_float8 *pix_f8, d_float8 *cmnParams_f8)
+{
+    pix_f8->f4[0] = rpp_hip_pixel_check_0to1((pix_f8->f4[0] - cmnParams_f8->f4[0]) * cmnParams_f8->f4[1] * (float4) ONE_OVER_255);
```

Comment:
@asalmanp : Can you verify if we really need this pixel check. Can we safely assume the output will be in 0-1 range?

Reply:
@rrawther we need to know the range of the pix_f8 and cmnParams_f8 to see if we can make the assumption that the output is between 0 to 1. @r-abishek can you verify the range of these variables?

Reply:
@rrawther @asalmanp All rpp tensor functions till now ensure all outputs for f16/f32 are in 0-1, for u8 are in 0-255 and i8 are in -128-127. Just for CMN, we see that this cannot be done (can't safely assume that the output will be between 0-1 since it does a (src - mean) / stdDev and the values could go out of range.

However, please note that in a few more changes being done to this PR, we will be removing the rpp_hip_pixel_check_0to1() for CMN since the math from the PyTorch training was verified and the outputs seem to do no range check. (So, output values are going outside the 0-1 range in PyTorch version). 

To match that, RPP will have to have an exception for just the CMN, that we don't check and saturate boundaries for all CMN functions. Please let us know any thoughts

Reply:
Ok. We need to analyze each function and see if the pixel range check is needed.